docs/how-to-guides/hello-world-example.md

```diff
@@ -7,131 +7,120 @@ import TabItem from '@theme/TabItem';
 
 This tutorial will demonstrate how to get started with Rancher Desktop by pushing an app to a local Kubernetes cluster.
 
-Rancher Desktop works with two container engines, [`containerd`](https://containerd.io/) and [Moby](https://mobyproject.org/), the open-sourced components of the Docker ecosystem. For `nerdctl`, use the containerd runtime. For `docker`, use the Moby (`dockerd`) runtime.
+Rancher Desktop works with two container engines, [containerd](https://containerd.io/) and [Moby](https://mobyproject.org/), the open-sourced components of the Docker ecosystem. For `nerdctl`, use the **containerd** runtime. For `docker`, use the **dockerd(moby)** runtime.
 
-### Hello World example
+### Example#1 - Build Image & Run Container
 
-**Create a folder:**
+#### Create a folder
 ```
 mkdir ../hello-world
 cd ../hello-world
 ```
 
-**Create a Dockerfile with the command below.**
+#### Create a Dockerfile with the command below
 ```
 FROM alpine  
 CMD ["echo", "Hello World!!"]
 ```
 
-**Build and run the image for verification purposes:**
+#### Build and run the image for verification purposes
 
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl build -t helloworld:v1.0 .
+nerdctl build --tag helloworld:v1.0 .
 nerdctl images | grep helloworld
 nerdctl run --rm helloworld:v1.0
-nerdctl rmi helloworld:v1.0 #To remove the image
+#To remove the image
```

Comment:
Nitpicking, but I think there should always be a space between `#` and the first character.
Also `To ` is just meaningless filler.
```suggestion
# Remove the image
```

Reply:
Modified per your suggestion.

Reply:
> Modified per your suggestion.

But not everywhere, and not here either. :)

Reply:
Done :)